docs/modules/reference/pages/service-assurance/monitors/MailTransportMonitor.adoc

```diff
@@ -282,30 +282,67 @@ Note that you must include the `monitor` section in your definition.
         </sendmail-test>
-        <readmail-test attempt-interval="5000" subject-match="OpenNMS Test Message" mail-folder="OPENNMS" debug="false" >
-          <readmail-host host="imap.gmail.com" port="993">
-            <readmail-protocol ssl-enable="true" start-tls="false" transport="imaps" />
+        <readmail-test attempt-interval="5000" <13>
+                       subject-match="OpenNMS Test Message" <14>
+                       mail-folder="OPENNMS" <15>
+                       debug="false" > <16>
+          <readmail-host host="imap.gmail.com" <17>
+                         port="993"> <18>
+            <readmail-protocol ssl-enable="true" <19>
+                               start-tls="false" <20>
+                               transport="imaps" /> <21>
           </readmail-host>
-          <user-auth user-name="opennms@gmail.com" password="opennms"/>
+          <user-auth user-name=opennms@gmail.com <11>
+                     password="opennms"/> <12>
         </readmail-test>
       </mail-test>
     </mail-transport-test>
   </parameter>
-  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-  <parameter key="ds-name" value="mtm_lat"/>
-  <parameter key="retry" value="20" />
+  <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <22>
+  <parameter key="ds-name" value="mtm_lat"/> <23>
+  <parameter key="retry" value="20" /> <24>
 </service>
 
-<monitor service="MTM" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <1>
+<monitor service="MTM" class-name="org.opennms.netmgt.poller.monitors.MailTransportMonitor" /> <25>
 ----
-<1> Required *monitor* section.
+<1> Interval in milliseconds between send attempts.
+<2> Whether to use authentication, in the event it is required.
+<3> Use the JavaMail Mail Transport Agent.
+<4> Show additional debug output.
+<5> The SMTP server address for sending mail.
+<6> The SMTP server port.
+<7> Use smtpsend or an alternate mailer.
+<8> The destination address.
+<9> The message subject.
+<10> The body of the message.
+<11> The user name for SMTP, POP, or IMAP authentication.
+<12> The password for SMTP, POP, or IMAP authentication.
+<13> Attempt to read email after this many milliseconds have passed; also used for retry interval.
+<14> Substring match in email subjects when looking for a specific email.
+<15> The folder or IMAP label to check for mail.
+<16> Show additional debug output.
+<17> The target host for reading mail.
+<18> The appropriate port for the protocol.
+<19> Whether to enable SSL for the connection.
+<20> Use the STARTTLS command
+<21> The transport protocol to use: pop3, pop3s, imap, or imaps.
+<22> Base directory of an RRD repository in which to store this service monitor’s response-time samples.
+<23> Name of the RRD data source (DS) name in which to store this service monitor’s response-time samples.
+<24> Number of attempts to test a service’s status
+<25> Required *monitor* section.
```

Comment:
Agree. This would be better in two columns. If we can't do columns, maybe a table that doesn't show the borders?